core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -851,6 +851,27 @@ public ExecResult execInContainer(String... command)
     }
 
     /**
+     *
+     * Method allow to copy file which we have in our repository to docker container
+     *
+     * @param localPath path with file which we would like to place in container
+     * @param containerPath path where we want to copy file
+     */
+    @Override
+    public void copyFileToContanier(String localPath, String containerPath){
+
+        if (!isRunning()) {
```

Comment:
since `isRunning` calls Docker API, I would remove this check and catch an exception from `.exec()`